style.css

```diff
@@ -0,0 +1,566 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Lato, sans-serif;
+}
+
+.container {
+  background-image: url('./images/base/mobile_intro_bg.png');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  padding: 1rem;
+}
+
+.desktop-nav {
+  display: none;
+}
+
+.nav-items a {
+  display: block;
+  text-decoration: none;
+}
+
+.logo {
+  display: none;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #272a33;
+}
+
+.burger {
+  position: relative;
+  cursor: pointer;
+  height: 4em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.menu-bar {
+  position: relative;
+  cursor: pointer;
+  height: 4em;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.burger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.burger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: #000;
+}
+
+.burger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: #000;
+}
+
+.heading {
+  margin-top: 3rem;
+  color: #ec5242;
+}
+
+.heading h1 {
+  font-size: 2.5rem;
+}
+
+.heading h2 {
+  /* color: #868686; */
+  margin-top: 1.875rem;
+  margin-bottom: 1rem;
+  color: #595959;
+  font-family: Gothic A1, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 900;
+  line-height: 0px; /* 0% */
+  letter-spacing: -1.6px;
+}
+
+.title {
+  font-size: 1.563rem;
+  font-weight: 300;
+}
+
+.heading .second-title {
+  color: #868686;
+  margin-top: 0.5rem;
+  padding-bottom: 1.563rem;
+}
+
+.heading .description {
+  margin-top: 2.313rem;
+  border: #fff 2px solid;
+  background-color: #fff;
+  padding: 1rem;
+  line-height: 1.25rem;
+  color: #868686;
+  display: inline-block;
+}
+
+/* main section */
+
+.main-program {
+  background: url('./images/base/program-summary-bcg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding: 1rem;
+}
+
+.main-program h1 {
+  text-align: center;
+  color: #fff;
+  font-weight: 400;
+  margin-bottom: 1.25rem;
+}
+
+.main-program h1::after {
+  content: '';
+  height: 2px;
+  width: 50px;
+  background-color: #ec5242;
+  bottom: 0;
+  border-radius: 10px;
+  margin: 0 auto;
+  display: block;
+  margin-top: 0.8em;
+}
+
+.card {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1rem;
+  background: rgba(255, 255, 255, 0.1);
+  width: 100%;
+  margin-bottom: 0.625rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.card-title {
+  color: #ec5242;
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+.card-info {
+  color: #fff;
+}

```

Comment:
- [ ] Good job so far, :clap: However, Please ensure that all your `card-info`  class properties are aligned properly to match the given design requirement, as it is right now your alignments are not matching, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. Please resolve it. :slightly_smiling_face: 

Below is an illustrations of yours and the given footer Design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student mobile main style](https://github.com/brighton0102/HTML-CSS-JavaScript-Capstone-Project/assets/43528594/ababaabf-0fec-45eb-918b-34e85364dc25)

</td>
      <td>

![Design mobile main styles](https://github.com/brighton0102/HTML-CSS-JavaScript-Capstone-Project/assets/43528594/a3ecba7d-85da-4ae4-a2ce-e34f6eb01ea9)


</td>
    </tr>
  </table>